server/server.js

```diff
@@ -148,6 +148,24 @@ module.exports = function(port, tweetSearcher, googleAuthoriser) {
         }
     });
 
+    app.put("/admin/administrators", function(req, res) {
+        try {
+            googleAuthoriser.addAdmin(req.body.email);
+            res.sendStatus(200);
+        } catch (err) {
+            res.sendStatus(404);
```

Comment:
We're not actually throwing an error on a bad write that we could check to see if 500 is more appropriate, just logging to the command line. How do I reject a promise with a specific reason?